traits/metricstrait/hack/servcieMonitor.yaml

```diff
@@ -0,0 +1,465 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.4
+  creationTimestamp: null
+  name: servicemonitors.monitoring.coreos.com
```

Comment:
why put this CRD in hack? And I didn't see any where using it?

Reply:
This is used in the controllers/suite_test.go, the test needs to be able to emit servcieMonitor using the mocking api-server

Reply:
I see, maybe putting it to a folder under test is better?